### PR TITLE
FIX: [exit] Use configuration instead of kine fixed interval

### PR DESCRIPTION
**Pull request by anywhy**

### CODE REVIEW

File: pkg/bbgo/exit_protective_stop_loss.go [hunk 1]
```diff
@@ -30,15 +30,19 @@ type ProtectiveStopLoss struct {
 	// PlaceStopOrder places the stop order on exchange and lock the balance
 	PlaceStopOrder bool `json:"placeStopOrder"`
 
+	// Interval is the time resolution to update the stop order
+	// KLine per Interval will be used for updating the stop order
+	Interval types.Interval `json:"interval,omitempty"`
```
Comment (c9s) [inline pkg/bbgo/exit_protective_stop_loss.go:35]: can you set the default value to 1m if it’s empty?

Comment (anywhy) [reply pkg/bbgo/exit_protective_stop_loss.go:35]: ok

**Comment by c9s** [comment]
@anywhy see this link https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Review by c9s — APPROVED** [review]

**Comment by c9s** [comment]
@anywhy i have approve it, you need to fix the cla signing :-) 

https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by c9s** [comment]
nvm, i used administrator mode to merge it

**Pull request merged**